CONTRIBUTING.md

```diff
@@ -1,300 +1,46 @@
-
-* We ask that you use the changelog system to document your change, this prevents our players from being caught unaware by changes - you can find more information about this here http://tgstation13.org/wiki/Guide_to_Changelogs
-
-* If you are proposing multiple changes, which change many different aspects of the code, you are expected to section them off into different pull requests in order to make it easier to review them and to deny/accept the changes that are deemed acceptable.
-
-* If your pull request is accepted, the code you add no longer belongs exclusively to you but to everyone; everyone is free to work on it, but you are also free to object to any changes being made, which will be noted by a Project Lead or Project Manager. It is a shame this has to be explicitly said, but there have been cases where this would've saved some trouble.
-
-* Please explain why you are submitting the pull request, and how you think your change will be beneficial to the game. Failure to do so will be grounds for rejecting the PR.
-
-## Banned content
-Do not add any of the following in a Pull Request or risk getting the PR closed:
-* National Socialist Party of Germany content, National Socialist Party of Germany related content, or National Socialist Party of Germany references
-* Code where one line of code is split across mutiple lines (except for multiple, separate strings and comments and in those cases existing longer lines must not be split up)
-
-##A word on git
-Yes we know that the files have a tonne of mixed windows and linux line endings, attempts to fix this have been met with less than stellar success and as such we have decided to give up caring until such a time as it matters.
-
-Therefore EOF settings of main repo are forbidden territory one must avoid wandering into
+Obey the following standards when contributing to the Github.
+
+#Things that should not be in your code or icon files:
+1. Backdoors for specific people
+2. if(prob(100))
+3. Anything that can kill anyone on the server in seconds
+4. Anything that can kill the server in seconds
+5. Anything directly copy pasted from a closed/leaked code base of SS13 (Goon for instance); permission granted nulls this rule.
+6. Anything that lags the server
+7. ERP
+8. Multiple "in world" calls in one proc
+9. Anything that expands its tile (32x32) unless its broken into 32x32 tiles
+10. Massively animated icons (We are talking very large animations])
+11. Anything that requires access to savefiles, bans, jobbans, notes, etc (unless noted)
+
+#Sound file rules
+1. .ogg is recommended
+2. .ogg compressed at 7
+3. If the sound has the same two stereo tracks; make it mono
+4. 22050HZ with a 16-bit PCM
+5. Midi files are allowed
+6. Ensure your soundfile is as small as possible; cut out parts if its music
+7. Title music doesn't require the previous rule
+
+#Do not mess with
+1. Anything in the config folder
```

Comment:
Which is redundant because the config files on the repo aren't the live ones, they're just templates. At the moment only @JamieH can update our config files